src/utility/validation/rules.stories.mdx

```diff
@@ -0,0 +1,39 @@
+import { Meta } from '@storybook/addon-docs/blocks'
+
+<Meta title="Examples|Validation" />
+
+# Form Validation
+
+`@system76/forms` includes form validation for inputs. It uses `vee-validate`
+behind the scenes, but exposes everything you should need in this package. There
+are a couple of benefits to this. To start, we have rewrote all of the error
```

Comment:
silly nitpick. "...we have rewritten all of the..." instead of "...we have rewrote all..."